dnaStreaming/listener.py

```diff
@@ -20,42 +18,55 @@ def __init__(self, service_account_id=None, user_id=None, client_id=None, passwo
     def _initialize(self, config):
         self.config = config
 
-    def listen(self, on_message_callback, maximum_messages=DEFAULT_UNLIMITED_MESSAGES, subscription_id=None):
-        limit_pull_calls = not (maximum_messages == self.DEFAULT_UNLIMITED_MESSAGES)
+    def listen(self, on_message_callback, maximum_messages=DEFAULT_UNLIMITED_MESSAGES, subscription_id="", batch_size=10):
         pubsub_client = pubsub_service.get_client(self.config)
 
-        subscription_id = subscription_id if subscription_id is not None else self.config.subscription()
-
-        if subscription_id is None or subscription_id.strip() == '':
+        subscription_id = subscription_id or self.config.subscription()
+        if not subscription_id.strip():
```

Comment:
potential NoneType error on the "strip()" if subscription_id is None, get rid of strip to prevent